code/datums/components/tackle.dm

```diff
@@ -0,0 +1,338 @@
+	the roll, the more severe the result.
+
+	Mod 1: Speed
+		-Base tackle speed is 1, which is what normal gripper gloves use. For other sources with higher speed tackles, like dolphin and ESPECIALLY rocket gloves, we obey Newton's laws and hit things harder.
+		-For every unit of speed above 1, move the lower bound of the roll up by 15. Unlike Mod 2, this only serves to raise the lower bound, so it can't be directly counteracted by anything you can control.
+
+	Mod 2: Misc
+		-Flat modifiers, these take whatever you rolled and add/subtract to it, with the end result capped between the minimum from Mod 1 and 100. Note that since we can't roll higher than 100 to start with,
+			wearing a helmet should be enough to remove any chance of permanently paralyzing yourself and dramatically lessen knocking yourself unconscious, even with rocket gloves. Will expand on maybe
+
+		Wearing a helmet: -5
+		Clumsy: +6
+
+	Effects: Below are the outcomes based off your roll, in order of increasing severity
+
+		1-63: Knocked down for a few seconds and a bit of brute and stamina damage
+		64-83: Knocked silly, gain some confusion as well as the above
+		84-93: Cranial trauma, get a concussion and go deaf for a bit, plus more damage
+		94-98: Knocked unconscious, get a concussion and a random mild brain trauma, as well as medium deafness and a fair amount of damage
+		99-100: Break your spinal cord, get paralyzed, take a bunch of damage too. Very unlucky!
+
+*/
+
+/datum/component/tackler/proc/splat(mob/living/carbon/user, atom/hit, datum/thrownthing/throwingdatum)
+	var/oopsie_mod = 0
+	var/danger_zone = (speed - 1) * 15 // for every extra speed we have over 1, take away 10 of the safest chance
+	danger_zone = max(min(danger_zone, 100), 1)
+
+	if(ishuman(user))
+		var/mob/living/carbon/human/S = user
+		var/head_slot = S.get_item_by_slot(ITEM_SLOT_HEAD)
+		if(head_slot && (istype(head_slot,/obj/item/clothing/head/helmet) || istype(head_slot,/obj/item/clothing/head/hardhat)))
+			oopsie_mod -= 5
+
+	if(HAS_TRAIT(user, TRAIT_CLUMSY))
+		oopsie_mod += 6 //honk!
+
+	var/oopsie = rand(danger_zone, 100)
+
+	switch(oopsie)
+		if(99 to 100)
+			// can you imagine standing around minding your own business when all of the sudden some guy fucking launches himself into a wall at full speed and irreparably paralyzes himself?
+			user.visible_message("<span class='danger'>[user] slams face-first into [hit] at an awkward angle, severing [user.p_their()] spinal column with a sickening crack! Holy shit!</span>", "<span class='userdanger'>You slam face-first into [hit] at an awkward angle, severing your spinal column with a sickening crack! Holy shit!</span>")
+			user.take_bodypart_damage(stamina=30, brute=40)
+			playsound(user, 'sound/effects/blobattack.ogg', 60, TRUE)
+			playsound(user, 'sound/effects/splat.ogg', 70, TRUE)
+			user.emote("scream")
+			user.gain_trauma(/datum/brain_trauma/severe/paralysis/paraplegic) // oopsie indeed!
```

Comment:
Paraplegic is TRAUMA_RESILIENCE_ABSOLUTE since it's made for quirks and thus incurable. If you want to force legs use the normal paralysis with the "legs" argument, or leave it random if you prefer.